wolfssl/ssl.h

```diff
@@ -748,12 +748,7 @@ enum AlertDescription {
     access_denied                   =  49,
     decode_error                    =  50,
     decrypt_error                   =  51,
-    #ifdef WOLFSSL_MYSQL_COMPATIBLE
-    /* catch name conflict for enum protocol with MYSQL build */
-    wc_protocol_version             =  70,
-    #else
-    protocol_version                =  70,
-    #endif
+    alert_protocol_version          =  70,
```

Comment:
I'm worried that this may break projects using `wc_protocol_version` or `protocol_version`. Please add a define defining this to `alert_protocol_version`.

Reply:
I think `protocol_version` can't be defined. 
This was to avoid some noisy ifdefs. 
Maybe we do some other preprocessor tricks to avoid that and to don't break current projects.

Reply:
I still think it would be safe to at least add a define somewhere for users using the old defines. Backwards compatibility is also important.

Reply:
In the end I added a define to maps to the correct enum